Dockerfile.test-runner

```
@@ -0,0 +1,31 @@
+    robotframework \
+    robotframework-seleniumlibrary \
+    robotframework-browser \
+    playwright
+
+# Initialize rfbrowser node dependencies and install Playwright Chromium.
+# This is required for Browser library tests that use: New Browser    chromium
+RUN rfbrowser init --skip-browsers && python -m playwright install chromium
+
+# Create optimized virtual display script using sh (smaller than bash)
+# Use printf to ensure Unix line endings
+RUN printf '#!/bin/sh\nif [ "$HEALING_SESSION" = "true" ]; then\n  export DISPLAY=:99\n  Xvfb :99 -screen 0 1920x1080x24 -ac +extension GLX +render -noreset &\n  sleep 1\nfi\nexec "$@"\n' > /usr/local/bin/start-with-display.sh && \
```

Hadolint (2.14.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)